doc/ext/math.rst

```diff
@@ -44,6 +44,14 @@ or use Python raw strings (``r"raw"``).
 
    Example: ``'Eq.{number}'`` is rendered as ``Eq.10``
 
+.. confval:: math_numfig
+
+   If ``True``, displayed math equations are numbered across pages in html and
+   related (epub, ...) output when :confval:`numfig` is enabled.
+   :confval:`numfig_secnum_depth` is respected.  The :rst:role:`eq` role must
+   be used to reference equation numbers, not the :rst:role:`numref` role.
+   Default is ``True``.
+
```

Comment:
I forgot to say a ``.. versionadded:: 1.7`` should be added

Reply:
done